app/src/NativeBase/Components/List.tsx

```diff
@@ -116,18 +130,65 @@ const List: FC<ListProps> = ({ data, mode, options, searchScreen, type }) => {
       return (
         <FlatList
           data={data as Projects}
-          keyExtractor={project => project.res_id}
           renderItem={({ item }) => {
             return (
-              <ProjectListItem
-                onPress={() => {
-                  navigate('ProjectDetail', { item, key: item.res_id })
-                }}
-              >
-                <ProjectSummary project={item} />
-              </ProjectListItem>
+              <Card margin="2">
+                <Pressable
+                  onPress={() => {
+                    navigate('ProjectDetail', { item, key: item.res_id })
+                  }}
+                  overflow="hidden"
+                >
+                  <VStack space={4}>
+                    <HStack
+                      justifyContent="space-between"
+                      alignItems="center"
+                      paddingRight="4"
+                    >
+                      <Heading width="70%" fontSize="sm">
+                        {item.name}
+                      </Heading>
+                      <ShareIcon
+                        size="md"
+                        color="accentGrey.100"
+                        onPress={underDevelopmentAlert}
+                      />
+                      <FavouriteIcon
+                        size="md"
+                        color="accentGrey.100"
+                        onPress={underDevelopmentAlert}
+                      />
+                    </HStack>
+
+                    <Text fontSize="xs">{item.client}</Text>
```

Comment:
It looks like there's much bigger vertical spacing above especially, but also below, the organisation name than on the Figma designs?  Not sure if it's to do with how the `VStack` spacing is working and/or some extra space inside the `HStack` above?

Reply:
I had padding in the VStack - looks better with it removed although there's one project which still has more spacing than the others...which is odd...

<img width="292" alt="Screenshot 2023-04-17 at 16 56 04" src="https://user-images.githubusercontent.com/87582782/232542519-dd2bee21-1e10-48e1-907b-6d21b403497b.png">

<img width="294" alt="Screenshot 2023-04-17 at 16 56 21" src="https://user-images.githubusercontent.com/87582782/232542528-d37803a1-c975-44a6-8e41-53f70239a5fe.png">

Reply:
Looking a lot better, thanks 👍   Might be good enough for now, unless you can easily figure out what's going on with that errant one.